frameworks/g-cloud-12/questions/services/serviceName.yml

```diff
@@ -14,6 +14,6 @@ type: text
 
 validations:
   - name: answer_required
-    message: You need to answer this question.
+    message: Enter the service name.
   - name: under_character_limit
```

Comment:
Query: are there 2 separate limitations for this field - word and character count? Could we limit it to word count only?